packages/common/test/pipes/number_pipe_spec.ts

```diff
@@ -141,6 +141,11 @@ import {beforeEach, describe, expect, it} from '@angular/core/testing/src/testin
           expect(pipe.transform(5.1234, 'DKK', '', '', 'da')).toEqual('5,12');
         });
 
+        it('should use the injected default currency code if none is provided', () => {
+		  const clpPipe = new CurrencyPipe('en-US', 'CLP');
+		  expect(clpPipe.transform(1234)).toEqual('1,234');
+		});
```

Comment:
The output actually contains the `CLP` as the currency.
Also we need to fix the indentation.

```suggestion
        it('should use the injected default currency code if none is provided', () => {
		      const clpPipe = new CurrencyPipe('en-US', 'CLP');
		      expect(clpPipe.transform(1234)).toEqual('CLP1,234');
		    });
```